dictionary.md

```diff
@@ -126,9 +146,23 @@ Gherkin is a domain-specific programming language created to write behavior desc
 
 ## H
 
+### Health Check
+
+<!-- TODO -->
+
 ### Host
 
-A set of [instances](#instance) installed on the single operating system (host) together with software that controls instances and allows communication with external elements of the system via protocols (e.g. http)
+A set of instances installed on the single operating system (host) together with software that controls instances and allows communication with external elements of the system via protocols (e.g. http)
+
+- host -  a source of information or signals. The term can refer to a computer, smartphone, tablet or any electronic device. In a network, clients (users' machines) and servers are hosts because they are both sources of information in contrast to network devices, such as routers and switches, which only direct traffic.
+
+- host program - software/program that runs in a computer that provides the source of information or signals.
+
+### Hub
+
+<!-- TODO - what is the difference between Hub and Host, we still use both terms.
```

Comment:
I thought that externally we use only words:
- STH/Transform-Hub for the open source STH.
- Hub - while describing the Host (as it is used in the code) in the cloud: the instance of class Host in the MultiHost process. However, "Host" is only an internal naming, while publicly it is branded as "Hub".

Isn't it so?

Reply:
true, the definition of Hub is unnecessary I guess. We will talk this over anyway 😉